schema/caveschema.xsd

```diff
@@ -57,7 +155,28 @@
             </xs:sequence>
           </xs:complexType>
         </xs:element>
-        <xs:element ref="Global" minOccurs="0" maxOccurs="1" />
+        <xs:element name="Global" maxOccurs="1">
+          <xs:annotation>
+            <xs:documentation>Global Settings</xs:documentation>
+          </xs:annotation>
+          <xs:complexType>
+            <xs:sequence>
+              <xs:element name="CameraPos" type="CameraType" />
+              <xs:element name="CaveCameraPos" type="CameraType" />
+              <xs:element name="Background">
+                <xs:complexType>
+                  <xs:attribute name="color" type="color" use="optional" default="0, 0, 0" />
```

Comment:
```suggestion
                  <xs:attribute name="color" type="color" default="0, 0, 0" />
```
My linter says that the `use="optional"` is redundant – would it be easier to drop it everywhere?

Reply:
Oh it must be because I'm giving a default, I'll remove

Reply:
Some of them don't define a default value so I'll leave the optional parts there